### PR TITLE
Allow passing action scripts to CRIU

### PR DESCRIPTION
**Pull request by ehmm**
Add option to lxc-checkpoint to allow passing of action script to CRIU from lxc-checkpoint command

### CODE REVIEW

**Comment by ehmm** [comment]
Added a signed off line

**Comment by stgraber** [comment]
Your commit list looks a bit confusing, let me rebase that for you :)

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
@brauner should be good to review

**Comment by brauner** [comment]
@tych0, I don't have objections. What do you think?

**Comment by tych0** [comment]
Yep, seems reasonable to me.

File: src/lxc/tools/lxc_checkpoint.c [hunk 1]
```diff
@@ -83,6 +85,11 @@ static int my_parser(struct lxc_arguments *args, int c, char *arg)
 		if (!checkpoint_dir)
 			return -1;
 		break;
+        case 'A':
+		actionscript_path = strdup(arg);
```
Comment (brauner) [inline src/lxc/tools/lxc_checkpoint.c]: You need to free this on error, don't you?

**Pull request closed without merging**